go/kbfs/libkbfs/block_retrieval_queue.go

```diff
@@ -266,41 +267,55 @@ func (brq *blockRetrievalQueue) notifyWorker(priority int) {
 	brq.sendWork(workerCh)
 }
 
-func (brq *blockRetrievalQueue) initPrefetchStatusCacheLocked() {
+func (brq *blockRetrievalQueue) initPrefetchStatusCacheLocked() error {
 	if brq.prefetchStatusForTest != nil {
-		return
+		return nil
 	}
 	if !brq.config.IsTestMode() && brq.config.Mode().Type() != InitSingleOp {
 		// If the disk block cache directory can't be accessed due to
 		// permission errors (happens sometimes on iOS for some
 		// reason), we might need to rely on this in-memory map.
-		// TODO(KBFS-3750): make it an LRU cache.
 		brq.log.Warning("No disk block cache is initialized when not testing")
 	}
 	brq.log.CDebugf(nil, "Using a local cache for prefetch status")
-	brq.prefetchStatusForTest = make(map[kbfsblock.ID]PrefetchStatus)
+	// TODO: choose a value for this size
+	var err error
+	brq.prefetchStatusForTest, err = lru.New(1000)
```

Comment:
Should we replace the `ForText` suffix with something more descriptive?

Reply:
`ForNoDiskCache`?

Reply:
@strib any thoughts on size here?

Reply:
The in-memory block cache uses 10000, so maybe you can copy that here?  It should probably be the same size.